network-store-iidm-impl/src/test/java/com/powsybl/network/store/iidm/impl/tck/extensions/RemoteReactivePowerControlTest.java

```diff
@@ -12,4 +12,13 @@
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
 public class RemoteReactivePowerControlTest extends AbstractRemoteReactivePowerControlTest {
+
+    @Override
+    public void terminalReplacementTest() {
+
+    }
+
+    public void replacementTest() {
```

Comment:
to check https://github.com/powsybl/powsybl-core/pull/3510

Reply:
Looks good now